tests/e2e/test_rosacli_cluster.go

```diff
@@ -854,6 +855,126 @@ var _ = Describe("Classic cluster creation validation",
 				Expect(err).NotTo(BeNil())
 				Expect(errorOutput.String()).To(ContainSubstring("etcd encryption cannot be disabled on clusters with FIPS mode"))
 			})
+
+		It("Create rosa cluster with additional security groups will validate well via rosacli - [id:68971]",
+			labels.Medium, labels.Runtime.Day1Negative,
+			func() {
+				var (
+					ocpVersionBelow4_14 = "4.13.12"
+					ocpVersion4_14      = "4.14.0"
+					index               int
+					flagName            string
+					securityGroups      = map[string]string{
+						"--additional-infra-security-group-ids":         "sg-aisgi",
+						"--additional-control-plane-security-group-ids": "sg-acpsgi",
+						"--additional-compute-security-group-ids":       "sg-acsgi",
+					}
+
+					invalidSecurityGroups = map[string]string{
+						"--additional-infra-security-group-ids":         "invalid",
+						"--additional-control-plane-security-group-ids": "invalid",
+						"--additional-compute-security-group-ids":       "invalid",
+					}
+				)
+				profile := profilehandler.LoadProfileYamlFileByENV()
+
+				By("Prepare creation command")
+				var command string
+				var rosalCommand config.Command
+				flags, err := profilehandler.GenerateClusterCreateFlags(profile, rosaClient)
+				Expect(err).To(BeNil())
+
+				command = "rosa create cluster --cluster-name " + profile.ClusterConfig.Name + " " + strings.Join(flags, " ")
+				rosalCommand = config.GenerateCommand(command)
```

Comment:
forgive my ignorance but what does the l stand for in this variable name? long command?

Reply:
@OAharoni-RedHat yes, 'l' is used being used to indicate its a long rosa command which is triggering cluster creation. several other test cases (for example OCP-38770)  were using the same var name, i thought to use same for consistency, Please let me know if it can be written in some other way to make it more readable. Thanks!

Reply:
Makes sense to me, I hadn't seen it before is all :). Thanks for the info!